lib/hexdocs/search/typesense.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Hexdocs.Search.Typesense do
+  require Logger
+
+  @behaviour Hexdocs.Search
+
+  @collection "hexdocs"
+
+  @impl true
+  def index(package, version, search_items) do
+    full_package = "#{package}-#{version}"
+
+    ndjson =
+      Enum.map(search_items, fn item ->
+        json =
+          Map.take(item, ["type", "ref", "title", "doc"])
+          |> Map.put("package", full_package)
+          |> Jason.encode_to_iodata!()
+
+        [json, ?\n]
+      end)
+
+    url = url("collections/#{@collection}/documents/import?action=create")
+    headers = headers([{"content-type", "text/plain"}])
```

Comment:
I was trying to change this content type value but no matter what I set, their API sends text/plain back as opposed to, ideally, application/x-ndjson. Was this the reason you put text/plain here? Any idea why they don't specify JSON-related content type in response?

Reply:
I copied the content type from the cURL example in the docs: https://typesense.org/docs/27.1/api/documents.html#index-multiple-documents, I guess it only accepts ndjson so content-type=text/plain can be removed. And the response content-type is hardcoded: https://github.com/typesense/typesense/blob/e9cc6abfb2eeb0c464ab64f35b608acf196a7d21/src/core_api.cpp#L1404

Reply:
Gotcha, thank you for digging this up.